integrations/access/msteams/config.go

```diff
@@ -22,18 +22,26 @@ import (
 
 	"github.com/gravitational/teleport/api/types"
 	"github.com/gravitational/teleport/integrations/access/common"
+	"github.com/gravitational/teleport/integrations/access/common/teleport"
 	"github.com/gravitational/teleport/integrations/access/msteams/msapi"
 	"github.com/gravitational/teleport/integrations/lib"
 	"github.com/gravitational/teleport/integrations/lib/logger"
 )
 
 // Config represents plugin configuration
 type Config struct {
+	common.BaseConfig
+
 	Teleport   lib.TeleportConfig
+	Client     teleport.Client
 	Recipients common.RawRecipientsMap `toml:"role_to_recipients"`
 	Log        logger.Config
 	MSAPI      msapi.Config `toml:"msapi"`
 	Preload    bool         `toml:"preload"`
+
+	// StatusSink receives any status updates from the plugin for
+	// further processing. Status updates will be ignored if not set.
+	StatusSink common.StatusSink
```

Comment:
`common.BaseConfig` also contains `Teleport`, `Client`, `Recipients` and `Log` fields. If the base config is embedded, those fields should not be redeclared on the teams app Config.